astropy/units/equivalencies.py

```diff
@@ -508,6 +508,59 @@ def convert_K_to_Jy(x_K):
     return [(astrophys.Jy, si.K, convert_Jy_to_K, convert_K_to_Jy)]
 
 
+def thermodynamic_temperature(disp, T_cmb=2.7255*si.K):
```

Comment:
I feel like the default value could be a proper constant.

Reply:
Agreed with @pllim - I think this is taken just from the Planck13 data, so I would write `T_cmb=None` here and then, if it is `None`, get the value from the Planck15 cosmology - best to do the import inside the function, so that cosmology only gets imported if needed.